basin3d/plugins/usgs.py

```diff
@@ -370,34 +402,32 @@ def list(self, query: QueryMonitoringFeature):
                         elif not query.monitoring_features:
                             yield monitoring_feature
 
-            # points: USGS calls these sites
             else:
-                if query.monitoring_features:
+                base_url = '{}site/?{}={}&seriesCatalogOutput=true&outputDataTypeCd=iv,dv&siteStatus=all&format=rdb'
+                # Points by id: USGS calls these sites
+                if query.monitoring_features is not None:
                     usgs_sites = ",".join(query.monitoring_features)
-                    feature_observed_properties = self.get_observed_properties_variables(query.monitoring_features)
+                    url = base_url.format(self.datasource.location, 'sites', usgs_sites)
                 else:
-                    # Get the variables with data
-                    feature_observed_properties = self.get_observed_properties_variables(usgs_subbasins)
-                    usgs_sites = ",".join(feature_observed_properties.keys())
+                    # Point by subbasin: USGS calls subbasin as huc (instead of sites) to retrieve all subbasins
+                    usgs_subbasin = ",".join(usgs_subbasins)
+                    url = base_url.format(self.datasource.location, 'huc', usgs_subbasin)
 
                 # Filter by locations with data
-                url = '{}site/?sites={}'.format(self.datasource.location, usgs_sites)
                 usgs_site_response = get_url(url)
                 logger.debug(f"{self.__class__.__name__}.list url:{url}")
 
                 if usgs_site_response and usgs_site_response.status_code == 200:
-
-                    for v in iter_rdb_to_json(usgs_site_response.text):
-                        yield _load_point_obj(datasource=self, json_obj=v, synthesis_messages=synthesis_messages,
-                                              feature_observed_properties=feature_observed_properties)
-
+                    observed_properties, unique_sites = parse_sites_response(usgs_site_response)
+                    for k,v in unique_sites.items():
```

Comment:
if the key is not used (doesn't look like it), you can use the following
`for v in unique_sites.values():`

Reply:
@dschristianson  Noted and changed